azurerm/data_source_kubernetes_cluster.go

```diff
@@ -77,6 +77,20 @@ func dataSourceArmKubernetesCluster() *schema.Resource {
 								},
 							},
 						},
+
+						"azure_policy": {
+							Type:     schema.TypeList,
+							MaxItems: 1,
+							Optional: true,
```

Comment:
Attributes in DataSources need to be `Computed: true`.  `MaxItems: 1` Can also be removed as it's only used when Creating/Updating a resource.

Reply:
Done